src/ui/pages/import.vue

```diff
@@ -132,18 +133,20 @@ const result = await Usecase.getCourses(ports)(
 );
 if (isResultError(result)) throw result;
 
-type CourseResult = {
-  course: DisplayCourse;
-  schedules: Schedule[];
-  selected: boolean;
-  expanded: boolean;
-};
+const registered = await ports.courseRepository.getRegisteredCoursesByYear(
```

Comment:
この方法ですと UI (Presentation 層) が Repository に依存してしまっており、現行のアーキテクチャと乖離してしまうため、以下のように UseCase を参照するのがいいと思いました。

```suggestion
const registered = await Usecase.getRegisteredCoursesByYear(
```